lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
Great, so I think we agree that we should try some additional new migrations to quash the issue, since I really wouldn't want to try to debug the migration error Eric lists above.